composer.json

```diff
@@ -14,5 +14,9 @@
             "type": "vcs",
             "url": "https://github.com/shopwareLabs/plugin-dev-tools"
         }
-    ]
+    ],
+  "require": {
```

Comment:
why is this necessary?

Reply:
What exactly? Adding those php extensions isn't strictly required but a recommended practice. My IDE is freaking out all the time when there are valid but not declared dependencies and composer will as well (if missing). This is therefore also already used in shopware's `composer.json`.